docs/framework/data/wcf/index.md

```diff
@@ -41,29 +41,29 @@ Just show me some code...
 
 I want to know more about OData...
 
-- [Whitepaper: Introducing OData](https://go.microsoft.com/fwlink/?LinkId=220867)
+- [Whitepaper: Introducing OData](../../../framework/wcf/index.md)
 
-- [Open Data Protocol Web site](https://go.microsoft.com/fwlink/?LinkID=184554)
+- [Open Data Protocol Web site](https://www.odata.org/documentation/)
 
-- [OData: SDK](https://go.microsoft.com/fwlink/?LinkID=185248)
+- [OData: SDK](https://www.odata.org/wp-content/uploads/sites/21/odatasdkcodesamples.zip)
 
-- [OData: Frequently Asked Questions](https://go.microsoft.com/fwlink/?LinkId=185867)
+- [OData: Frequently Asked Questions](https://www.odata.org/faq)
 
 I want to watch some videos...
 
-- [Beginner's Guide to WCF Data Services](https://go.microsoft.com/fwlink/?LinkId=220864)
+- [Beginner's Guide to WCF Data Services](https://docs.microsoft.com/dotnet/framework/wcf/index)
 
-- [WCF Data Services Developer Videos](https://go.microsoft.com/fwlink/?LinkId=220861)
+- [WCF Data Services Developer Videos](https://docs.microsoft.com/sql/connect/sql-data-developer&view=sql-server-ver15#dataservices)
```

Comment:
Not really a good target. Probably the old location was removed and simply redirected to that

```suggestion
```